server/common/database/helpers.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const inRange = require('lodash/inRange');
+const notEmpty = input => input.length > 0;
+
+module.exports = {
+  alterEnum,
+  getValidator,
+  setLogging
+};
+
+// NOTE: Enables safe altering of ENUM values: https://git.io/fxzeS
+function alterEnum(queryInterface, Sequelize, table, column, options = {}) {
```

Comment:
Please rather use sequelize-replace-enum-postgres since it does the same thing if values are added but also handles value removal.

Reply:
TBH we don't need it at all. I put it there because we might want use it in future so I'll simply remove it. I'm not sure I follow you, this should handle value removal too? :thinking: